### DISCUSSION TITLE
Trivy Office Hours 04.01.23

### DISCUSSION THREAD

**Discussion by AnaisUrlichs** [aquasecurity/trivy#3372] [Office Hours]
Join us for the first Trivy Office Hours in the new year! We highly encourage community members to get involved, ask questions and showcase their use of Trivy and use cases.

**Registration Link:** https://aquasec.zoom.us/meeting/register/tJApdOGprjwtH9MypxCF2RaVAaOVWpOjwXuz

**Agenda:**
- Introduction from Giovanni
- Trivy Release v0.36.0 Update
- Trivy-Operator Updates

Please comment below anything that you would like to cover, we will then prioritise and put it onto the agenda. 

If you would like to present a use case or similar for Trivy or related projects, please keep the presentation to 10min max and clearly state in your comment if you want to present your proposed agenda item. (Presentation could just be a short demo or verbally sharing how you use the projects.)

Note that this call will be recorded and shared on the [Aqua Open Source YouTube channel.](https://www.youtube.com/@aquasecoss)

The Trivy Office Hours follow our [Code of Conduct](https://github.com/aquasecurity/community/blob/main/CODE_OF_CONDUCT.md), please be respectful to each other.

You can find more information on the [Community GitHub repository](https://github.com/aquasecurity/community/blob/main/office-hours/trivy-office-hours.md).

**Comment by ingentismg** [comment]
There is a typo in the link to the YouTube channel. It does not use an @.

\Update: Fixed
